main.py

```diff
@@ -103,7 +103,7 @@ def configure_confluent_kafka_consumer(event, args):
 
     # PUT payload if connectors do exist this updates existing connector
 
-    if existing_connector in existing_connectors:
+    if existing_connectors in existing_connectors:
```

Comment:
This was correct originally.

Reply:
needs to change to `if args.connector_name in existing_connectors:`
This is already done